wrappers/python/pyrsutil.cpp

```diff
@@ -4,6 +4,8 @@ Copyright(c) 2017 Intel Corporation. All Rights Reserved. */
 #include "python.hpp"
 #include "../include/librealsense2/rsutil.h"
 
+#include <regex>
```

Comment:
why is this needed?

Reply:
Good question, I can't remember why I add this line.
Will remove

Reply:
ok